packages/openactive-integration-tests/test/features/payment/prepayment-optional/feature.json

```diff
@@ -1,10 +1,11 @@
 {
-  "category": "advanced-payment",
+  "category": "payment",
   "identifier": "prepayment-optional",
   "name": "prepayment optional",
   "description": "Support for booking with optional payment",
   "explainer": "",
   "specificationReference": "https://www.openactive.io/open-booking-api/EditorsDraft/#booking-without-payment",
   "required": false,
-  "coverageStatus": "none"
+  "coverageStatus": "none",
```

Comment:
note to self: Why is `coverageStatus: none`? Probably just a mistake

Reply:
yes, agreed, likely a mistake